packages/ra-ui-materialui/src/layout/LoadingIndicator.tsx

```diff
@@ -16,7 +16,12 @@ export const LoadingIndicator = (props: LoadingIndicatorProps) => {
     const theme = useTheme();
     return (
         <Root className={className} sx={sx}>
-            {loading ? (
+            <RefreshIconButton
```

Comment:
Why?

Reply:
LoadingIndicator needs to replace the circular arrow icon by a circular loader, and it needs to do it precisely for the loading effect to be nice. The current way of doing it didn't work with a different `spacing` preference in the theme, and couldn't be adjusted in the theme using component `styleOverrides`. 

The new way of showing the circular loader is more robust, can be overridden, and adapts well to smaller font sizes (see the Nano theme).